src/main/content/_assets/js/builds.js

```diff
@@ -88,6 +94,23 @@ function getAllowedBuilds(build_type, package_locations, liberty_version) {
     });
 }
 
+/**
+ * Return URL corresponding to the key
+ * @param {String} sig_key - e.g. "kernal.sig"
+ */
+function getURLForSig(list, sig_key) {
+    if(!list) {
+        return list;
```

Comment:
It's a little unclear while looking at this function why we would want to return an empty value if nothing is passed in. Maybe if we add in the comments that it's optional.

Reply:
Can you comment on if https://github.com/OpenLiberty/openliberty.io/pull/2441/commits/8d6b84c1bbc8298b50b75417ef9ba2f6014778cd has improved the readability of the code?